src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj

```diff
@@ -4,8 +4,10 @@
     <PackageId>Microsoft.Azure.Management.Network</PackageId>
     <Description>Provides management capabilities for Network services.</Description>
     <AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-    <Version>14.1.0-preview</Version>
-    <PackageTags>Microsoft Azure Network management;Network;Network management;windowsazureofficial</PackageTags>    
+    <Version>14.2.0-preview</Version>
+    <PackageTags>Microsoft Azure Network management;Network;Network management;windowsazureofficial</PackageTags>
+    <PackageReleaseNotes>This is a preview release of the Network SDKs. It includes the support for application security groups (ASGs), secure VNETs, ILB HA,
```

Comment:
Network SDK is one of the top 5 RPs. Providing details that helps to understand what the new version has is very helpful for developers who are trying to develop using Netwok SDK.
What does ILB HA mean?